raiden_installer/tokens.py

```diff
@@ -208,6 +212,30 @@ class TokensV33(Enum):
     WIZ = _WizardToken
 
 
+class TokensV36(Enum):
+    RDN = Erc20Token(
+        ticker="RDN",
+        wei_ticker="REI",
+        addresses={
+            "goerli": "0x4074fD4d460d0c31cbEdC3f59B2D98626D063952",
+        },
+    )
+    SAI = _SAI
```

Comment:
Can we somehow change SAI everywhere to DAI?

Reply:
I think SAI is a specific token that serves exactly one purpose, like the `_LondonRDN` Token.
So I'm pretty sure that shouldn't be changed. I also didn't want to delete it because I have no clue what it is/was used for.